docs/docs/examples/workflow/router_query_engine.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #82.                    response_generated.append(query_engine.query(query))
</pre><p>Should use async</p><p> </p>

<pre class="ql-syntax" spellcheck="false">tasks = []
for selected_query_engine in selected_query_engines.selections:
&nbsp; &nbsp; print(
&nbsp; &nbsp; &nbsp; &nbsp; f"Selected query engine: {selected_query_engine.index}: {selected_query_engine.reason}"
&nbsp; &nbsp; )
&nbsp; &nbsp; query_engine = query_engines[selected_query_engine.index]
&nbsp; &nbsp; tasks.append(query_engine.aquery(query))

response_generated = await asyncio.gather(*tasks)

</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/15635/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>